es/index.html

```diff
@@ -14,68 +14,57 @@
 </head>
 <body>
   <main>
-    <h1>No pidas permiso para preguntar, sólo pregunta.</h1>
+    <h1>No preguntes por preguntar, sólo pregunta</h1>
     <p>
-      De vez en cuando, en las salas de chat en las que participo,
-      alguien se conecta y pregunta algo similar a,
+      De vez en cuando, en las salas de chat en línea en las que me muevo, alguien entra y dice algo parecido a,
     </p>
     <blockquote>
       <span class="name">Foobar123:</span>
       <p class="message">
-        ¿hay alguien que sepa de Java?
+        Hay algún experto en Java por aquí?
       </p>
     </blockquote>
     <p>
-      Esto está mal por muchas razones. 
-      Lo que esta persona está <em>en realidad</em> preguntando es,
+      Esto es una mala forma de preguntar, por varias razones. Lo que la persona realmente está preguntando aquí es,
     </p>
     <blockquote>
       <span class="name">Foobar123:</span>
       <p class="message">
-        ¿hay alguien que sepa de Java que quiera ceder a mirar mi problema,
-        sea lo que sea,
-        incluso si al final resulta que no tiene que ver con Java,
-        o incluso si alguien que no sepa de Java supiese la respuesta a mi pregunta?
+        ¿Hay algún experto en Java por aquí que esté dispuesto a comprometerse a estudiar mi problema, sea cual sea,
+        incluso si no está relacionado con Java o si alguien que no sabe nada de Java puede responder a mi pregunta?
       </p>
     </blockquote>
     <p>
-      Hay un montón de razones por las que alguien que SÍ sepa la respuesta ya no va a responder.
-      Por preguntar esto,
-      estás pidiendo más de lo que crees que estás pidiendo.
+      Hay muchas razones por las que la gente que SÍ tiene los conocimientos no los admitirá.
+      Preguntando, estás pidiendo más de lo que crees que estás pidiendo.
     </p>
     <p>
-      Estás pidiendo a alguien tomar responsabilidad.
-      Estás cuestionando la confianza que otras personas tienen en sus propias capacidades.
-      También estás dejando fuera a otras personas de forma innecesaria.
-      Muchas veces respondo preguntas que tienen que ver con lenguajes o librerías que nunca he usado,
-      porque las respuestas son (de algún modo para una persona que desarrolle) de sentido común.
+      Estás pidiéndole a la gente que asuma responsabilidades. Estás cuestionando la confianza de la gente sobre sus habilidades.
+      También estás excluyendo innecesariamente a otras personas. A menudo respondo a preguntas relacionadas con lenguajes o bibliotecas que nunca he utilizado,
+      porque las respuestas son (en el sentido de un programador) de sentido común.
     </p>
     <p>
-      También puede ser visto como...
+      También puede ser visto como..
```

Comment:
I think that this is actually a typo in the English version, which indeed is using two dots. However, the proper way to do an ellipsis in Spanish (and in other languages as well after checking that /fr, /de, /it do it this way too) is with three dots, so I do not understand how this sentence improves with this change.

```suggestion
      También puede ser visto como...
```